l3kernel/testfiles/m3intarray001.luatex.tlg

```diff
@@ -114,11 +75,11 @@ TEST 5: Zeroing
 ============================================================
 0,0,24,0,0,0,0,28,0,0
 The integer array \g_testd_intarray contains 10 items:
->  0, 0, 24, 0, 0, 0, 0, 28, 0, 0.
+> 0, 0, 24, 0, 0, 0, 0, 28, 0, 0.
 24
 0,0,0,0,0,0,0,20,0,0
 The integer array \g_testd_intarray contains 10 items:
->  0, 0, 0, 0, 0, 0, 0, 20, 0, 0.
+> 0, 0, 0, 0, 0, 0, 0, 20, 0, 0.
```

Comment:
Ideally the extra space would be avoided in the non-Lua case?

Reply:
I agree and have a patch for that, but it's a bit hacky and I wanted to keep it separate from this PR since it's rather unrelated.

Reply:
Sounds good.